Assets/MixedRealityToolkit.Providers/XRSDK/XRSDKSubsystemHelpers.cs

```diff
@@ -36,5 +36,51 @@ public static XRInputSubsystem InputSubsystem
                 return inputSubsystem;
             }
         }
+
+        private static XRMeshSubsystem meshSubsystem = null;
+
+        /// <summary>
+        /// The XR SDK mesh subsystem for the currently loaded XR plug-in.
+        /// </summary>
+        public static XRMeshSubsystem MeshSubsystem
+        {
+            get
+            {
+#if XR_MANAGEMENT_ENABLED
+                if (meshSubsystem == null &&
+                    XRGeneralSettings.Instance != null &&
```

Comment:
Nit: Consider wrapping a private thing like 

ActiveLoader GetActiveLoader() 
{
    if (XRGeneralSettings.Instance != null &&
        XRGeneralSettings.Instance.Manager != null &&
        XRGeneralSettings.Instance.Manager.activeLoader != null)
        {
                    return XRGeneralSettings.Instance.Manager.activeLoader
        }
}

Or maybe

Reply:
Or maybe not

Reply:
Updated!